types/index.d.ts

```diff
@@ -85,9 +85,7 @@ declare module '@azure/functions' {
         /**
          * HTTP response object. Provided to your function when using HTTP Bindings.
          */
-        res?: {
```

Comment:
I don't see this change happening any time soon. Unless you disagree, I think that means this should be tracked as an issue instead of a PR